ui/temboardui/web/routes/inventory.py

```diff
@@ -163,7 +163,8 @@ def put_instance(address=None, port=None, instance=None):
             g.db_session.execute(instance.disable_plugin(plugin))
     for plugin in new_plugins - current_plugins:
         if plugin not in current_app.temboard.plugins:
-            flask.abort(400, f"Unknown plugin {plugin}.")
+            logger.error("Unknown plugin %s", plugin)
```

Comment:
Ok, je comprends le cas. 

Effectivement, si un greffon existe dans l'agent mais pas dans l'UI, on peut simplement l'ignorer. logger.debug suffit. Message: `Plugin not enabled on UI: %s`.